src/app-gocardless/bank-factory.js

```diff
@@ -59,6 +60,7 @@ export const banks = [
   SpkWormsAlzeyRiedMalade51wor,
   SwedbankHabaLV22,
   VirginNrnbgb22,
+  HanseaticBank,
```

Comment:
Same here

Reply:
Ah sorry, totally missed that that is in alphabetical order